qiskit_dynamics/models/frame.py

```diff
@@ -625,6 +641,10 @@ def _conjugate_and_add(
         Note: B is added in the frame basis before any potential final change
         out of the frame basis.
         """
+        if self.vectorized_operators:
+            # faster than the flatten operation later.
```

Comment:
What does this comment mean?

Reply:
Ah–the reshape (n**2,) -> (n,n) is about three orders of magnitude faster than the reshape (n,n) -> (n**2,) in the tests I've done. I can remove the comments, if they're not clear.